### ISSUE TITLE
How to detect on an image using C# language after loaded model

### ISSUE THREAD

**Issue by smartyfn** [migueldeicaza/TensorFlowSharp#359]
I am a C# engineer. My project uses keras in Python to detect image. I know tensorflowsharp but it doesn't have C# api of keras. How to detect on an image using C# language in keras environment. Any suggestions will be greatly appreciated.

**Comment by sdg002** [comment]
Hi ,
I am not a contributing author to this project. But, I did develop a sample project which demonstrates training a CNN using Python to recognize MNIST digits and then detect the test pictures using TensorFlowSharp C# library.

If this is what you are looking for then , this sample code is hyperlinked here:
[https://github.com/migueldeicaza/TensorFlowSharp/issues/347](MNIST - From Python to C# using TensorFlowSharp)

This is a very simple C# console application. All the hardwork is done by TensorFlow and TensorFlowSharp

Cheers

**Comment by oferbentovim** [comment]
here is a (not mine)
https://engineering.matterport.com/splash-of-color-instance-segmentation-with-mask-r-cnn-and-tensorflow-7c761e238b46
